build.sbt

```diff
@@ -1650,109 +1617,6 @@ lazy val runtime = (project in file("engine/runtime"))
       necessaryModules ++ langs ++ tools
     }
   )
-  .settings(
```

Comment:
I was hoping the setup of `runtime` project will be simpler. Removal of these lines matches my expectations.